# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../colordiff/colordiff-1.0.9-xmlto-skip-validation.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Do not attempt to validate the input before processing

This is a workaround to get the man pages generated, because they do not
validate correctly.

--- colordiff-1.0.9/Makefile.orig	2011-05-11 12:07:50.776002080 +0200
+++ colordiff-1.0.9/Makefile	2011-05-11 12:08:33.900002073 +0200
@@ -10,10 +10,10 @@
 
 
 doc: colordiff.xml cdiff.xml
-	xmlto -vv man colordiff.xml
-	xmlto -vv man cdiff.xml
-	xmlto -vv txt colordiff.xml
-	xmlto -vv html-nochunks colordiff.xml
+	xmlto -vv --skip-validation man colordiff.xml
+	xmlto -vv --skip-validation man cdiff.xml
+	xmlto -vv --skip-validation txt colordiff.xml
+	xmlto -vv --skip-validation html-nochunks colordiff.xml
 	mv colordiff.txt README
 	perl -p -i -e 's#<head>#<head><link rel=\"stylesheet\" type=\"text/css\" href=\"colordiff.css\">#' colordiff.html
 	perl -p -i -e 's#</body>#</div></body>#' colordiff.html
